src/Traits/HasRoles.php

```diff
@@ -277,4 +277,27 @@ protected function getStoredRole($role)
 
         return $role;
     }
+
+    /**
+     * Return all the permissions the user have via his roles.
+     *
+     * @return Collection
+     */
+    protected function getPermissionsViaRolesAttribute()
```

Comment:
It felt better to have these as regular function and not as accessors. These functions are mentioned in the readme.